mirrord/cli/src/execution.rs

```diff
@@ -162,9 +162,12 @@ impl MirrordExecution {
         if let Ok(v) = std::env::var(INJECTION_ENV_VAR) {
             env_vars.insert(INJECTION_ENV_VAR.to_string(), format!("{v}:{lib_path}"))
         } else {
-            env_vars.insert(INJECTION_ENV_VAR.to_string(), lib_path)
+            env_vars.insert(INJECTION_ENV_VAR.to_string(), lib_path.clone())
         };
 
+        // for use in execvp (restoring library path)
+        env_vars.insert("MIRRORD_LIBRARY_PATH".to_string(), lib_path);
```

Comment:
Is `MIRRORD_LIBRARY_PATH` used anywhere? If not I think we can undo the changes to this file.

Reply:
Nice catch.